wordpress/package.json

```diff
@@ -9,8 +9,8 @@
   "private": true,
   "scripts": {
     "start:assets": "NEXT_PUBLIC_IS_THIS_NEXT=false NODE_ENV=development webpack server",
-    "start:wp": "THEME_NAME=superstack PROJECT_CODE=spck CONTAINER_NAME=spck_wp ./scripts/mutagen.sh up && docker-compose exec -T wp bash < ./scripts/provision.sh",
-    "stop:wp": "./scripts/mutagen.sh down",
+    "start:wp": "sh ./scripts/start-wp.sh",
```

Comment:
@kuuak In the `provision.sh` file, which is now called in the newly created `start-wp.sh` file, we need the `THEME_NAME` variable. We were setting it in the `start:wp` script previously. Don't you think we should still set it here instead of changing the `provision.sh` file on a specific project ? I don't know if there's maybe an other way to set it only at one place ?

Reply:
Ok, sounds good ! We should maybe update the Notion "Setup a new project [v3]" where we were updating the `THEME_NAME` and `PROJECT_CODE` values

Reply:
@charl0tee I updated our internal doc

![Capture d’écran 2024-07-30 à 11 13 30](https://github.com/user-attachments/assets/53c3fc13-521e-44e2-b150-6152b120a051)